frontend/src/components/eMIB/EmibIntroductionPage.jsx

```diff
@@ -11,7 +20,23 @@ const styles = {
 
 class EmibIntroductionPage extends Component {
   static propTypes = {
-    nextPage: PropTypes.func.isRequired
+    nextPage: PropTypes.func.isRequired,
+    // Props from Redux
+    updateEmailsEnState: PropTypes.func,
+    updateEmailsFrState: PropTypes.func,
+    updateEmailsState: PropTypes.func,
+    getTestQuestions: PropTypes.func
+  };
+
+  componentDidMount = () => {
```

Comment:
These emails should not be loaded until hitting "start test". Otherwise, they'd have access to questions in the console before the test actually starts.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/375/commits/0a9342fb04a8c5885d425ac48580af5138091efe and commit #https://github.com/code-for-canada/project-thundercat/pull/375/commits/79325287397df436ccd52a2e85d2aa0e2a2eec73.